src/prefect/utilities/logging.py

```diff
@@ -22,14 +22,18 @@ class CloudHandler(logging.StreamHandler):
     def __init__(self) -> None:
         super().__init__()
         self.client = None
-        self.errored_out = False
+        self.logger = logger = logging.getLogger("CloudHandler")
```

Comment:
Why the extra `= logger =` here? I don't think the variable is used anywhere

Reply:
haha total oversight!

Reply:
Updated.